ui/app/AppLayouts/CommunitiesPortal/CommunitiesPortalLayout.qml

```diff
@@ -207,8 +208,59 @@ ScrollView {
             anchors.centerIn: parent
             store: root.communitiesStore
             onClosed: {
+                root.communitiesStore.clearDiscordCategoriesAndChannels()
                 destroy()
             }
         }
     }
+
+    Component {
+        id: chooseCommunityCreationTypePopupComponent
+        StatusModal {
+          id: chooseCommunityCreationTypePopup
+          anchors.centerIn: parent
+          showHeader: false
+          onClosed: {
+              destroy()
+          }
+
+          Item {
+              id: content
+              anchors.fill: parent
+              implicitHeight: chooseCommunityCreationTypeContent.implicitHeight
+              ColumnLayout {
+                  id: chooseCommunityCreationTypeContent
+                  anchors.horizontalCenter: parent.horizontalCenter
+
+                  spacing: 24
+
+                  Item {
+                      width: parent.width
+                      height: 20
+                  }
+
+                  StatusButton {
+                      text: qsTr("Create new Status Community")
+                      Layout.alignment: Qt.AlignHCenter
+                      onClicked: {
+                          chooseCommunityCreationTypePopup.close()
+                          Global.openPopup(createCommunitiesPopupComponent)
+                      }
+                  }
+
+                  StatusButton {
+                      text: qsTr("Import a community from Discord")
+                      Layout.alignment: Qt.AlignHCenter
+                      onClicked: {
+                          chooseCommunityCreationTypePopup.close()
+                          Global.openPopup(createCommunitiesPopupComponent, {
+                            stackTitle: qsTr("Import a community from Discord into Status"),
+                            finishButtonLabel: qsTr("Finalise Discord import")
+                          })
```

Comment:
Here, we also need to find a way to extend `createCommunitiesPopupComponent`'s stack children as we need two more stack items when it's opened from here.

I had a brief discussion with @MishkaRogachev about that. Probably will investigate a solution around these lines: https://stackoverflow.com/questions/57304987/adding-dynamically-to-qml-tabbar-and-stacklayout